src/generic-provider-views/index.js

```diff
@@ -339,14 +369,17 @@ module.exports = class View {
       addFile: this.addFile,
       filterItems: this.filterItems,
       filterQuery: this.filterQuery,
+      toggleSearch: this.toggleSearch,
       sortByTitle: this.sortByTitle,
       sortByDate: this.sortByDate,
       logout: this.logout,
       demo: this.plugin.opts.demo,
       isActiveRow: this.isActiveRow,
       getItemName: this.plugin.getItemName,
       getItemIcon: this.plugin.getItemIcon,
-      handleScroll: this.handleScroll
+      handleScroll: this.handleScroll,
+      title: this.plugin.title,
+      defaultViewType: this.opts.defaultViewType
```

Comment:
should we just call this `viewType`? Seems more apt that `defaultViewType`. The default view type might be 'list', but regardless of what this value is overridden to, the attribute name remains.

*I'm not sure if I articulated this well, but hopefully you get what I mean*
 🙂